code-challenges.test.js

```diff
@@ -14,39 +14,113 @@
 
 // a) Create a test with an expect statement using the variable provided.
 
-const hitchhikersCharacters = [
-  { name: "ford prefect", occupation: "a hitchhiker" },
-  { name: "zaphod beeblebrox", occupation: "president of the galaxy" },
-  { name: "arthur dent", occupation: "a radio employee" }
-]
-// Expected output: ["Ford Prefect is a hitchhiker.", "Zaphod Beeblebrox is president of the galaxy.", "Arthur Dent is a radio employee."]
+describe('generateSentences', () => {
+  const hitchhikersCharacters = [
+    { name: "ford prefect", occupation: "a hitchhiker" },
+    { name: "zaphod beeblebrox", occupation: "president of the galaxy" },
+    { name: "arthur dent", occupation: "a radio employee" }
+  ]
+
+  const expectedOutput = [
+    "Ford Prefect is a hitchhiker.",
+    "Zaphod Beeblebrox is president of the galaxy.",
+    "Arthur Dent is a radio employee."
+  ]
+
+  const result = generateSentences(hitchhikersCharacters)
+
+  it('takes in an array of objects and returns an array with a sentence about each person with their name capitalized', () => {
+    expect(result).toEqual(expectedOutput)
+  })
+
+  result.forEach((sentence, index) => {
```

Comment:
This part is not necessary in your test, and is more than likely why you were having issues with your test.